.github/workflows/kubernetes-deploy.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
             --set gateway.application.limits.keepClusterOnComplete=false \
             .
           GATEWAY=$(kubectl get pod -l app.kubernetes.io/name=gateway -o name)
-          kubectl wait --for=condition=Ready "$GATEWAY" --timeout 5m
+          kubectl wait --for=condition=Ready "$GATEWAY" --timeout 7m
```

Comment:
I increased a little bit the numbers because it was generating some false negatives the test.